### DISCUSSION TITLE
device.disableSynchronization() works for iOS tests but not Android

### DISCUSSION THREAD

**Discussion by iMaupin** [wix/Detox#2949] [Q&A]
I have some crucial tests for my app that require synchronization to be disabled. This command functions wonderfully when I run the tests on the iOS simulator. When I run the same tests on android my tests hang where they should desynchronize. Is there any known cause/solution for this?

**Comment by d4vidi** [comment]
@iMaupin please refer to our [`18.18.0` release](https://github.com/wix/Detox/releases/tag/18.18.0) notes.
